### DISCUSSION TITLE
Customise Confirmation Email

### DISCUSSION THREAD

**Discussion by justynpride** [HiEventsDev/Hi.Events#213] [Ideas]
This is part, idea, part, I think something isn't working. I've love to be able to customise the booking confirmation email a bit more, so that a lot of the key information that like to include is there. It makes the event more personal and ensure attendees get all the information upfront. In my mind this is a new feature. 

I did find in Settings the Email & Notification Settings option for setting text for the Email Footer. From what I have tested and seen, this only applies to messages sent via the message option on attendees and orders, and the text doesn't get included in the order confirmation email. I'm happy to have missed something on that.

**Comment by daveearley** [comment]
Good suggestion! I like the idea of giving people the ability to customize outgoing emails. It would work nicely with a WYSIWYG editor that allowed you to drop in variables like `{{attendee_name}}`.

There's an [existing ticket](https://github.com/orgs/HiEventsDev/projects/1/views/1?query=sort%3Aupdated-desc+is%3Aopen&pane=issue&itemId=74563421) in the backlog for this.

**Reply by justynpride** [reply]
Fantastic. I don't think I've given the Projects area enough of a review.
